boa/examples/closures.rs

```diff
@@ -0,0 +1,19 @@
+use boa::{Context, JsString};
+
+fn main() {
+    let mut context = Context::new();
+
+    let variable = JsString::new("I am a captured variable");
+
+    context
+        .register_global_closure("closure", 0, move |_, _, _| {
```

Comment:
I think i would be useful to let people know what the 3 arguments are that we don't need here